src/test/objectstore/store_test.cc

```diff
@@ -1317,6 +1317,99 @@ TEST_P(StoreTest, SimpleObjectTest) {
 
 #if defined(WITH_BLUESTORE)
 
+TEST_P(StoreTestSpecificAUSize, ReproBug41901Test) {
+  if(string(GetParam()) != "bluestore")
+    return;
+  SetVal(g_conf(), "bluestore_debug_enforce_settings", "hdd");
```

Comment:
nautilus does not have the "bluestore_debug_enforce_settings" option, which is causing failures like http://qa-proxy.ceph.com/teuthology/yuriw-2020-04-30_22:32:53-rados-wip-yuri3-testing-2020-04-30-1557-nautilus-distro-basic-smithi/5003166/teuthology.log